features/step_definitions/cli_steps.js

```diff
@@ -53,12 +58,12 @@ export default function cliSteps () {
 
       // replace placeholder for generate unique ids
       const replacedFileContent = fileContent
-        .replace(/\<id\-(\w)\>/gi, (match, g1) => uniqueId(g1))
+        .replace(/<id-(\w)>/gi, (match, g1) => uniqueId(g1))
 
-      mkdirp(dirName, err1 => {
+      mkdirp(dirName, (err1) => {
```

Comment:
err1 > directoryWriteError
err2 > fileWriteError
Maybe? Doing this for the linter I assume?